src/shared/ui/date-lang/date-lang.tsx

```diff
@@ -1,31 +1,33 @@
-/* eslint-disable @stylistic/jsx-one-expression-per-line */
+import classNames from 'classnames/bind';
 import micIcon from '@/shared/assets/icons/mic.svg';
 import noteIcon from '@/shared/assets/icons/note-icon.svg';
+import { getCourseDate } from '@/shared/helpers/getCourseDate';
 import Image from '@/shared/ui/image';
 
 import styles from './date-lang.module.scss';
 
+const cx = classNames.bind(styles);
+
 interface DateLangProps {
   startDate: string;
   mode: string;
   language: string[];
-  type?: string;
 }
 
-export const DateLang = ({ startDate, language, mode, type = '' }: DateLangProps) => {
-  const splittedLanguage = language.slice().join('/');
-
+export const DateLang = ({ startDate, language, mode }: DateLangProps) => {
   return (
-    <div className={`${styles.info} ${styles[type]}`}>
-      <p className={styles.date}>
-        <Image className={styles.note} src={noteIcon} alt="note-icon" lazy="false" />
-        <span>Start date {startDate ?? 'Not set'}</span>
+    <section className={cx('info')}>
+      <p className={cx('date')}>
+        <Image className={cx('icon')} src={noteIcon} alt="note-icon" lazy="false" />
+        <span>Start date:</span>
+        <time>{getCourseDate(startDate)}</time>
```

Comment:
I haven't seen the `time` tag before, but it seems like we should set the `datetime` attribute when using it
https://www.sitepoint.com/html5-time-element-guide/